### PR TITLE
Feature/set variables - support Zeebe 0.17

### PR DESCRIPTION
**Pull request by jwulf**
This pull request updates the client library to support the 0.17 release of Zeebe. It is a breaking change against the 0.16 server.

After merging, a new major package version should be published to npm, via merging to master and tagging master with 2.0.0. (Remember to push the merge to master before pushing the 2.0.0 tag!)

* Rename payload -> variables
* Rename ZBClient method `updateWorkflowInstancePayload` -> `setVariables`
* Update gRPC proto file to match Zeebe 0.17 gRPC methods
* Add integration tests
* Update README.md

### CODE REVIEW

**Comment by s3than** [comment]
Confirmed CI passes

**Comment by s3than** [comment]
Docs update has changed the package name from CreditSenseAU to jwulf

**Comment by jwulf** [comment]
Looks like it is auto-generated from the repo URL. Because I'm working in my fork, it is generating it based on that.

Two ways to redress that:

* Make me a committer in this organisation;
* Someone with committer access to this repo re-generate the docs and PR them in on top of this. @Jfilmer could do it. Just need to check-out, run `npm run docs`, then roll the PR.

**Comment by jwulf** [comment]
Yes, the typedoc-plugin-sourcefile-url package works to rewrite the links.

**Comment by jwulf** [comment]
One of the tests fails randomly. Looks like a race condition.

File: docs/classes/bpmnparser.html [hunk 1]
```diff
@@ -240,7 +240,7 @@ <h3>
 											<li>
 												Defined in
 												<a
-													href="https://github.com/CreditSenseAU/zeebe-client-node-js/blob/46ff7a6/src/lib/BpmnParser.ts#L43"
+													href="https://github.org/CreditSenseAU/lib/BpmnParser.ts#L43"
```
Comment (s3than) [inline docs/classes/bpmnparser.html]: github.com

Comment (jwulf) [reply docs/classes/bpmnparser.html]: Thanks for picking this up. Pushed this change.

**Pull request merged**